Events/SupervisorChanged.md

```diff
@@ -0,0 +1,61 @@
+# Supervisor Changed Event
+
+<table align="center">
+  <tr>
+    <td>Description</td>
+    <td>The HR system will raise an event for all subordinates of a person when they change, or for the person themself if it is known that their supervisor has changed.<br>Because employee job information is not public, a secur callback URL is provided for business applications that are authorized.</td>
```

Comment:
```suggestion
    <td>The HR system will raise an event for all subordinates of a person when they change, or for the person themself if it is known that their supervisor has changed.<br>Because employee job information is not public, a secure callback URL is provided for business applications that are authorized.</td>
```